microcosm_eventsource/immutable_event_factory.py

```diff
@@ -0,0 +1,99 @@
+"""
+Factory for immutable events.
```

Comment:
Nit: You changed the name of the underlying factory, but didn't change the name of this file/comments.